index.html

```diff
@@ -560,15 +564,18 @@
         <div dfn-for="MediaTrackSettings">
           <p>
             The <dfn><code>focalLength</code></dfn> dictionary member
-            represents the <a>depth map</a>'s <a>focal length</a>.
+            represents the <a>depth map</a>'s <a>focal length</a>. It is a
+            <a>non-constrainable property</a>.
```

Comment:
I'd be happy to mirror it, if educated how to do so the right way. There were no hooks to related concepts that could be referenced, also the facingMode prose seemed not reusable, consider:

>This string (or each string, when a list) should be one of the members of VideoFacingModeEnum. The members describe the directions that the camera can face, as seen from the user's perspective. Note that getConstraints may not return exactly the same string for strings not in this enum. This preserves the possibility of using a future version of WebIDL enum for this property.

... and:

>A camera can report multiple facing modes. For example, in a high-end telepresence solution with several cameras facing the user, a camera to the left of the user can report both "left" and "user".

Concrete suggestions welcome!